minecraft/src/main/java/org/spicord/velocity/VelocityJDADetector.java

```diff
@@ -0,0 +1,24 @@
+package org.spicord.velocity;
+
+import java.net.URL;
+
+import com.velocitypowered.api.plugin.PluginContainer;
+
+public class VelocityJDADetector {
+
+    public static void checkOtherJDA(SpicordVelocity spicordPlugin) {
+        for (PluginContainer plugin : SpicordVelocity.getProxyServer().getPluginManager().getPlugins()) {
+            if (plugin == spicordPlugin) {
+                continue;
+            }
+
+            ClassLoader classLoader = plugin.getClass().getClassLoader();
```

Comment:
You are obtaining the ClassLoader of the PluginContainer class, which is the ClassLoader of Velocity itself, not the ClassLoader which loaded the plugin.